nixos/modules/services/databases/riak-cs.nix

```diff
@@ -110,13 +118,6 @@ in
         '';
       };
 
-      extraAdvancedConfig = mkOption {
```

Comment:
It's not needed, because all options except for `key_perm` and `cert_perm` can be covered in the other configuration file (`riak-cs.conf`). The `advanced.conf` was a relic from Riak CS 1.x (mentioned in their docs) and is currently being replaced by `riak-cs.conf`. The only options that `riak-cs.conf` can't cover are the ones implemented in the options provided here and `key_perm` and `cert_perm`. `key_perm` and `cert_perm` are coming in another PR.